### PR TITLE
remove container injection in service

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/341

File: BackofficeBundle/DisplayIcon/DisplayManager.php [hunk 1]
```diff
@@ -2,23 +2,14 @@
...
 namespace OpenOrchestra\BackofficeBundle\DisplayIcon;
 
-use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAware;
 
 /**
  * Class DisplayIconManager
  */
-class DisplayManager
+class DisplayManager extends ContainerAware
```
Comment (itkg-ngilain) [inline BackofficeBundle/DisplayIcon/DisplayManager.php]: Can't you only inject the templating service here ?

File: BackofficeBundle/Twig/LeftPanelExtension.php [hunk 2]
```diff
@@ -2,24 +2,17 @@
...
 
-use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Class LeftPanelExtension
  */
-class LeftPanelExtension extends \Twig_Extension
+class LeftPanelExtension extends \Twig_Extension implements ContainerAwareInterface
```
Comment (itkg-ngilain) [inline BackofficeBundle/Twig/LeftPanelExtension.php]: Can't you instead inject the open_orchestra_backoffice.left_panel_manager service ?

File: BackofficeBundle/DisplayIcon/DisplayManager.php (hunk 1 above)
Comment (nicolasThal) [inline BackofficeBundle/DisplayIcon/DisplayManager.php]: I don't think so, there will be an injection loop

File: BackofficeBundle/Twig/LeftPanelExtension.php (hunk 2 above)
Comment (nicolasThal) [inline BackofficeBundle/Twig/LeftPanelExtension.php]: same, there is an injection loop

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**